geodepy/tests/test_transform.py

```diff
@@ -118,6 +127,38 @@ def test_atrftogda2020(self):
         assert (abs(alic_gda2020_comp[1] - alic_gda2020[1]) < 5e-5)
         assert (abs(alic_gda2020_comp[2] - alic_gda2020[2]) < 5e-5)
 
+    def test_mga94to20(self):
+        alic_mga94 = (53, 386352.3979, 7381850.7689, 603.3466)
```

Comment:
Where did you get the MGA94 coordinates for Alice? I've checked everything else against the GDA2020 Tech Manual and between both tests for consistency. I can't see them in the GDA94 Tech Manual

Reply:
I couldn't find any myself, so the grid coordinates have been transformed from the geographic coordinates (GDA94 and GDA2020 separately) shown in the Tech Manual example (sect. 3.1.1). Ellipsoid heights are as shown in the example.

Reply:
OK, I've already checked the ellipsoidal heights against 3.1.1 and I got the MGA2020 coordinates from D1.3, and they check out with yours. I used our GDA94 Redfearn's spreadsheet to convert the geographic coords given in 3.1.1 to grid coords and they match. 

So all looks good.